code-with-comment.html

```diff
@@ -21,8 +22,9 @@
       }
     </style>
 
-    <code-highlighter code="{{code}}" before-start="{{beforeStart}}" after-start="{{afterStart}}" can-split="true"></code-highlighter>
-    <comment-container hidden="[[hideComments]]" hide-add="{{hideAddComments}}" comments="{{comments}}"></comment-container>
+    <code-highlighter code="{{code}}" before-start="{{beforeStart}}" after-start="{{afterStart}}" can-split="true" amount-lines="{{amountLines}}"></code-highlighter>
```

Comment:
How can we do this then, so that `amountLines` is updated as it changes in code-highlighter?